.github/workflows/release.yml

```
@@ -11,18 +11,19 @@ jobs:
 
   test:
     name: run tests and linters
-    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rust.yml@v1
+    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rust.yml@v2.0.3
```

```suggestion
    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rust.yml@v2.0.4
```

let's consume `v2.0.4`, which contains [this fix](https://github.com/kubewarden/github-actions/pull/60) so the job can pass.

(the same for the other actions being used from kubewarden/github-actions)